src/runner.ts

```diff
@@ -380,46 +396,13 @@ export class ResolveRunner implements Types.IResolveRunner {
                 : error.path;
 
               if (errorPathInResult && errorPathInResult.length) {
-                _set(lookupResult.resolved.result, errorPathInResult, val);
+                set(lookupResult.resolved.result, errorPathInResult, val);
               } else if (lookupResult.resolved.result) {
                 lookupResult.resolved.result = val;
               }
             }
           }
         }
-
-        // support custom parsers
-        if (this.parseAuthorityResult) {
-          try {
-            // TODO: rework this to pass in an addValidation function to allow custom parsers to add their own validations
-            // then generally re-work the error system here to be based around more flexible validations
-            const parsed = await this.parseAuthorityResult({
-              authorityResult: lookupResult,
-              result: lookupResult.resolved.result,
-              targetAuthority: ref,
-              parentAuthority: this.authority,
-              parentPath,
-            });
-
-            // if (parsed.errors) {
-            // TODO: as mentioned above, allow caller to add errors/validations
-            // }
-
-            lookupResult.resolved.result = parsed.result;
-          } catch (e) {
-            // could not parse... roll back to original value
-            lookupResult.resolved.result = val;
-
-            lookupResult.error = {
```

Comment:
Shouldn't we leave this error out?

Reply:
The error is still captured as part of the `RESOLVE_AUTHORITY` error (see change in tests related to this).

Reply:
Yeah I saw the test was updated, but what if the parseAuthority function throws? Shouldn't we set the former error?